pkg/platform/runtime/store/store.go

```diff
@@ -255,7 +255,12 @@ func (s *Store) InstallPath() string {
 	return s.installPath
 }
 
+var ErrNoBuildPlanFile = errs.New("no build plan file")
+
 func (s *Store) BuildPlanRaw() ([]byte, error) {
+	if !fileutils.FileExists(s.buildPlanFile()) {
+		return nil, ErrNoBuildPlanFile
```

Comment:
The previous error would contain an IO error, which this does not. I'm not sure if that matters (ie. do we asser this anywhere?), but please verify that this isn't breaking anything.

Reply:
It's not. The only other callers end up logging the errors, not interpreting them in any other way.